Documentation/v2/admin_guide.md

```diff
@@ -202,10 +205,7 @@ $ tar -xzvf infra1.etcd.tar.gz -C %data_dir%
 ```
 
 ```
-etcd -name infra1 \
--listen-peer-urls http://10.0.1.13:2380 \
--listen-client-urls http://10.0.1.13:2379,http://127.0.0.1:2379 \
--advertise-client-urls http://10.0.1.13:2379,http://127.0.0.1:2379
+systemctl start etcd2
```

Comment:
Same as the `systemctl stop etcd2` part; have a separate codeblock for the systemctl example?